src/pages/PlacesPage/Placelist/Placelist.ts

```diff
@@ -10,22 +10,13 @@ import { Sight } from 'src/types/api';
 * @class
 */
 class Placelist extends Base {
-
-  async renderPlaces(sights: Sight[]) {
-    const placelist = document.getElementById('list-places') as HTMLDivElement;
-    sights.forEach((data) => new Place(placelist, data).render());
-  }
-    
-    
-
-  /**
-  * Рендерит список мест в DOM и запрашивает данные мест.
-  */
   async render() {
     await this.preRender();
 
-    get('sights')
-      .then((response) => this.renderPlaces(response.data.sights));
+    const response = await get('sights') as WithResponse<{ sights: Sight[] }>;
```

Comment:
Повторы по коду увидел (два раза точно): уже заслуживает вынесения в отдельный тип)
```suggestion
    // types/api.ts
    type Sights = {
        sights: Sight[];
    }
    
    const response = await get('sights') as WithResponse<Sights>;
```

Reply:
поправила